e2e/test/scenarios/dashboard/dashboard.cy.spec.js

```diff
@@ -178,6 +182,18 @@ describe("scenarios > dashboard", () => {
     cy.findByText("Orders, Count");
   });
 
+  it("should trigger window.onbeforeunload when trying to leave an unsaved, edited dashboard", () => {
```

Comment:
Not sure how to test this but the current doesn't fail if we remove the change in the code. Could we improve the test so it fails if there is no message?